docs/Acceptable-Formulae.md

```diff
@@ -29,7 +29,7 @@ Additionally, [homebrew/core](https://github.com/Homebrew/homebrew-core) formula
 
 - macOS
 - Java
-- OSXFuse
+- macFUSE
```

Comment:
This whole section from `Additionally` can actually be removed as it's now outdated.